Utilities/Source/Config/Config.h

```diff
@@ -0,0 +1,66 @@
+#ifndef CONFIG_H
+#define CONFIG_H
+
+#include <tuple>
+
+
+#define CAST(x) static_cast<std::size_t>(x)
```

Comment:
**(controversial):** i guess it better to change this and replace define.

Reply:
May be, may be. But which one is more readable:
`CONFIG.get<CAST(Para::NAME)>();`
or
`CONFIG.get<static_cast<std::size_t>(Para::NAME)>();`
?

Reply:
In second one it is more clear what cast you're doing. And also we declared that we do not use "defines" and "using namespaces" so it is maybe harder and longer to write code but easier to read code

Reply:
![image](https://user-images.githubusercontent.com/21951872/37934332-81b0b556-3156-11e8-810d-5a6a46bd7f18.png)

Where macros usage prohibition is mentioned in the picture (xwiki isn't available now)?

Reply:
You can try to find macros usage prohibition [here](https://github.com/Vasar007/FANUC/blob/master/CODESTYLE.md).